### PR TITLE
Add test coverage for __init__.py

### PR DESCRIPTION
**Pull request by major**
Signed-off-by: Major Hayden <major@mhtx.net>

### CODE REVIEW

File: tests/test_init.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    def test_parse_bad_patchwork_url(self):
+        """Ensure parse_patchwork_url() handles a parsing exception."""
+        patchwork_url = "garbage"
+        with self.assertRaises(Exception) as context:
+            skt.parse_patchwork_url(patchwork_url)
+
+            self.assertTrue(
+                "Can't parse patchwork url: '{}'".format(patchwork_url)
+                in context.exception
+            )
```
Comment (spbnick) [inline tests/test_init.py]: Hmm, I'm not quite getting how this works. Should this really be inside `with`?

Comment (major) [reply tests/test_init.py]: That's the only spot were the `context` variable is available to me.

**Comment by major** [comment]
@spbnick In my last software project(s), the full stop was required. I will do my best to remember it here! :)

**Comment by spbnick** [comment]
@major, full stops are a matter of aesthetics mostly, but I already started documenting the code this way :)

File: tests/test_init.py (hunk 1 above)
Comment (spbnick) [reply tests/test_init.py]: Hmm, then what would happen if the assertion checking throws the same exception, and why does [this code in the documentation](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises) has the test outside? Sorry, my context manager-fu is weak.

Comment (major) [reply tests/test_init.py]: Weird, it does work outside the `with:`. I removed the indention in the latest push.

**Comment by major** [comment]
Should be fixed in 887a8ab, @spbnick.

**Comment by spbnick** [comment]
Thank you, Major, merged!

**Pull request merged**